.eslintrc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = { // eslint-disable-line no-undef
     env: {
+        node: true,
```

Comment:
This is a little surprising, was something correcting oddly without it?

Reply:
Without this it would say that `process` does not exist, for the check if the build is production or development in html/js/index.js. Not sure if this was the correct solution, at least it was a solution ;).